src/components/current-query/index.js

```diff
@@ -35,6 +35,7 @@ class CurrentQuery extends React.Component {
         ));
 
       case "range-facet":
+      case "date-range-facet":
```

Comment:
This just lets the date range facet show up in the current query UI the way that the range facet does.

Reply:
@qgan7125 -- this is the part where we map the component type to the actual component to show